examples/sm/test_sm.cu

```diff
@@ -67,7 +67,7 @@ struct main_struct {
         !quiet);
 
     // counts of matched subgraphs
-    VertexT *ref_subgraph_match = new VertexT[data_graph.nodes];
+    unsigned long *ref_subgraph_match = new unsigned long[data_graph.nodes];
```

Comment:
Please change this back to `VertexT`.

Reply:
Because the corresponding variable on the GPU has to be unsigned long type, here I cannot change back to VertexT.